js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts

```diff
@@ -341,6 +341,7 @@ export enum OpenInferenceSpanKind {
   RERANKER = "reranker",
   EMBEDDING = "embedding",
   AGENT = "agent",
+  GUARDRAILS = "guardrails",
```

Comment:
```suggestion
  GUARDRAIL = "guardrail",
```
Can we make it singular? Also what would be the typical contents of a guardrail span?